Cargo.toml

```diff
@@ -21,6 +21,7 @@ members = [
     "examples/hacspec-sha3",
     "examples/hacspec-ntru-prime",
     "examples-unsafe",
+    "examples/hacspec-riot-bootloader",
```

Comment:
Since everything in here is hacspec I guess we can drop the hacspec prefix for the directory (keeping it only for the crate name).